datafusion/sqllogictest/test_files/groupby.slt

```diff
@@ -2278,6 +2278,27 @@ CREATE TABLE exchange_rates (
     (3, '2022-01-02 12:00:00'::timestamp, 'TRY', 'USD', 0.11),
     (4, '2022-01-03 10:00:00'::timestamp, 'EUR', 'USD', 1.12)
 
+# test count and count distinct  group by date_bin/date_part
+query error
+select date_bin(interval '1 minute', ts) as bin, count(distinct currency_from) as count from exchange_rates group by bin;
+
+query error
+select date_part('year', ts) as year, count(distinct currency_from) as count from exchange_rates group by year;
```

Comment:
These 2 tests should pass after this PR https://github.com/apache/arrow-datafusion/pull/8176 is merged